test/AutoRest.TestServer.Tests/Common/StringExtensionsTests.cs

```diff
@@ -53,6 +53,8 @@ public void ValidateSingularize(string noun, string expected, bool inputIsKnownT
         [TestCase("IPAddress", new[] { "IP", "Address" })]
         [TestCase("HTTPIsURL", new[] { "HTTP", "Is", "URL" })]
         [TestCase("GetAllByLocation", new[] { "Get", "All", "By", "Location" })]
+        [TestCase("snake_case", new[] { "Snake", "Case" })]
+        [TestCase("single", new[] { "Single" })]
```

Comment:
Can we add unit tests to verify all the different scenarios of List to Get?

Reply:
@ShivangiReja, could you confirm whether your question has been answered?

Reply:
Yes, now this PR looks good to me.